packages/okta-react/src/react-router/index.js

```diff
@@ -10,17 +10,12 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-import { Component } from 'react';
-import PropTypes from 'prop-types';
+import RouterSecurity from './RouterSecurity';
```

Comment:
Probably can try
```
export { default as RouterSecurity } from './RouterSecurity';
```

Reply:
We could, but what would be the benefit?  They get to type more?

Reply:
not really, it can end up with only 3 lines in the index file.

Reply:
I'm not seeing the benefit of adopting an unusual syntax that doesn't add clarity for the sake of a few lines.

Reply:
It's just a FYI comment. It has been adopted by some popular open source repos.

Reply:
oh?  Any examples?  I'm stumped as to why this syntax would be more popular, so examples can help.

Reply:
I learned it from `material-ui` repo. Here is an example https://github.com/mui-org/material-ui/blob/master/packages/material-ui/src/AppBar/index.js